ansible_collections/lvrfrc87/git_acp/plugins/module_utils/git_actions.py

```diff
@@ -247,6 +303,9 @@ def push(self):
 
         if push_force:
             command.append("--force")
+
+        if tag:
+            command.append("--tags")
```

Comment:
`--tags` does only push the tags. I don't think that's a problem tho since tasks in my understanding should be as "small" as possible. So you shouldn't really push changes AND tags in the same task.

Maybe there's a way to specify that certain parameter can't be used if certain others are given. That would throw an error in the ansible playbook and would force us to keep those tasks as simple as possible

Reply:
This works as intended, mutual exclusion of parameters will be implemented though.